### DISCUSSION TITLE
Issue enabling auto highlighting in MVTLayer via PyDeck

### DISCUSSION THREAD

**Discussion by diehl** [visgl/deck.gl#7775] [Q&A]
I’m currently trying to get auto highlighting to work in PyDeck for an MVTLayer and am having no luck. Here’s [the example I’m working with](https://colab.research.google.com/drive/1yx91WiXVe80a034J_o_6ThuYzKzO1aHP?usp=sharing), hosted on Colab. Am I missing something obvious here? Appreciate any pointers!

**Comment by kopp** [comment]
Does https://github.com/visgl/deck.gl/discussions/6732#discussioncomment-2297421 (the answer and the comments in the answer) solve your problem?

**Reply by diehl** [reply]
@kopp It did indeed help me answer the question ultimately. Looking through my notes, I wrote the following down that captured how I ultimately solved the problem in the data generation process. 

When saving a GeoPandas GeoDataFrame to disk as GeoJSON with `.to_file`, feature IDs will not be included and there is no option available to create `feature.id` fields. Exporting the GeoJSON in the following manner will create the feature IDs needed. 
```
with open('<GEOJSON FILENAME>', 'w') as f:
	f.write(gdf.to_json())

**Comment by ibgreen** [comment]
Perhaps we should consider adding findings like this to a FAQ to the pydeck docs? @ajduberstein

**Reply by diehl** [reply]
@ibgreen If you create a section like that, I'm happy to share findings that I've been making note of along the way.

**Reply by kopp** [reply]
Please also consider adding (and answering😉) https://github.com/visgl/deck.gl/discussions/8032 to such a collection.
